definitions/types/cidr.json

```diff
@@ -1,7 +1,7 @@
 {
   "$schema": "http://json-schema.org/draft-07/schema",
   "type": "string",
-  "pattern": "^([0-9]{1,3}\\.){3}[0-9]{1,3}(\/([0-9]|[1-2][0-9]|3[0-2]))?$",
+  "pattern": "^(?:[0-9]|[0-9]{2}|1[0-9][0-9]|2[0-4][0-9]|25[0-5])(?:\\.(?:[0-9]|[0-9]{2}|1[0-9][0-9]|2[0-4][0-9]|25[0-5])){3}(?:\/(?:[1-9]|1[0-9]|2[0-9]|3[0-2]))?$",
```

Comment:
Technically the only one would all `999.999.999.999` which is invalid. This ensures all octets are 0-255.